Gopkg.lock

```diff
@@ -2,185 +2,153 @@
 
 
 [[projects]]
-  digest = "1:2b10b9a545074605403d32baf9dda24b7582976ba7e9b46c4c7b9da9edac03e7"
   name = "cloud.google.com/go"
```

Comment:
It looks like the Gopkg.lock file was generated using the older dep v0.4. The lock file is now being maintained using dep v0.5.0. Please update your dep for consistency.